### ISSUE TITLE
Drops don't seem to be claimed

### ISSUE THREAD

**Issue by simonk83** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#354]
**Describe the bug**
Drop progress does increase (although as far as I can tell you won't see any drop progress in the logs unless you physically navigate to the stream in a browser, then it'll immediately show up in the logs, so that's probably a separate bug?) but the drops don't seem to be auto claimed

**To Reproduce**
Sea of Thieves drops are currently enabled for the next couple of days which is probably a good time to test this as I think you only need to link your microsoft account

**Expected behavior**
Drops should be auto claimed once the percentage hits 100%

**Desktop (please complete the following information):**
 - OS: Debian
 - Python version [e.g. 3.x] 3.8.10

**Additional context**

**Comment by simonk83** [comment]
Hmm. I just tried again with a different drop and this one worked fine. The difference, I guess, is that the Sea of Thieves drop only required 20 mins (and no progress showed up in the log for that), whereas this new one (Goose Goose Duck) required 60 mins and everything worked fine, including the log. Not sure if there's a minimum time in the code or something like that?

**Comment by simonk83** [comment]
And again with the latest SoT drop, no progress listed in the log and no drop claimed once it was ready

**Comment by Tkd-Alex** [comment]
Please share you logs file with debug level

**Comment by simonk83** [comment]
Yep will do, just need to wait for a short drop and enable debug then I'll get that to you

**Comment by simonk83** [comment]
Ok, didn't realise debug was on by default, so here's the log from the other day (it's quite big, sorry)

https://www.dropbox.com/s/757bg39zdf07bqi/BitMeIntoParts.log.2021-12-03.txt

It's the Sea of Thieves drops that didn't register or claim

**Comment by isinteger** [comment]
This has also happened to me.

**Comment by bricep** [comment]
This is happening to me as well. The drops occur every 15 minutes but are only seemingly being found or claimed every 25 minutes or so.
